requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
+torch
+torchvision
```

Comment:
Do we need to add torchvision to requirements? It seems to be only used in the examples.

Reply:
Yes, it's for convenience. Another way is keeping a "requirements_test.txt" for all deps, then "requirements.txt" keeps only core deps. Then users might need to run additional `pip install torchvision == some version` before run example. Your opinion?

Reply:
I think it is good for convenience in current status.